src/mito2/src/config.rs

```diff
@@ -65,6 +65,15 @@ pub struct MitoConfig {
     /// Number of meta action updated to trigger a new checkpoint
     /// for the manifest (default 10).
     pub manifest_checkpoint_distance: u64,
+    /// Number of removed files to keep in manifest's `removed_files` field before also
+    /// remove them from `removed_files`. Mostly for debugging purpose.
+    pub experimental_manifest_keep_removed_file_count: usize,
```

Comment:
Do we need experimental prefix?

Reply:
> Do we need experimental prefix?

yes? I mean this option could very possible change later when need to change how the manifest's `removed_files` field is updated?